packages/react-components/react-radio/src/components/RadioGroup/useRadioGroup.ts

```diff
@@ -18,7 +25,34 @@ export const useRadioGroup_unstable = (props: RadioGroupProps, ref: React.Ref<HT
 
   const generatedName = useId('radiogroup-');
 
-  const { name = generatedName, value, defaultValue, disabled, layout = 'vertical', onChange, required } = props;
+  const {
+    name = generatedName,
+    value,
+    defaultValue,
+    disabled,
+    imperativeRef,
+    layout = 'vertical',
+    onChange,
+    required,
+  } = props;
+
+  const rootRef = useMergedRefs(ref);
+
+  React.useImperativeHandle(
+    imperativeRef,
+    () => ({
+      focusSelected: () => {
```

Comment:
Could you please add a documentation example for this? If we're gonna make it public, let's explain it properly to people :)